build/Jenkinsfile.pr

```diff
@@ -81,7 +81,6 @@ pipeline {
                     ansiColor('xterm') {
                         sh """
                             cd /go/src/github.com/mattermost/mattermost-server
-                            mv config/default.json config/config.json || echo ""
```

Comment:
This was done for making Tests work in CI. My code copies config/default.json to temp.
This was breaking tests since config/default.json was not present during test setup.

Do you want me to change the code to look for config/default.json first if it's not there then look for config/config.json?

Reply:
Ah, interesting -- thanks for the clear explanation. We do need to preserve the effect of this `mv` in CI for the final tar bundle delivered to end users (no `default.json` and `config.json` initialized from it).

With my recent config refactoring, we no longer even rely on `default.json` to initialize the config -- if it's missing, we just use `config.SetDefaults()` and write out `config.json` accordingly.

Perhaps we could adopt the same approach here? Instead of copying `default.json` to `config.json`, just special case that file and write out the result of serializing `config.SetDefaults` to disk?